docs/src/_guide/advanced-topics.md

```diff
@@ -13,3 +13,159 @@ Most of Meltano's features are available without installing any additional packa
+
+### Do I need to do anything different with state?
+
+Nope! Meltano manages state for you and you can use any command you normally would with Singer-based connectors.
+
+### I have an Airbyte pipeline already running, how can I transfer it to Meltano?
+
+Transferring a pipeline to Meltano would require determining your strategy for handling incremental state and overall table structure.
+
+Because with Meltano you would be using a different loader (aka destination or target) than you would in your Airbyte pipeline, it’s likely that the table structure that would appear in your destination would be different. Due to this difference, we have two recommendations for migrating a pipeline.
+
+The first may be possible only if your data source allows you to quickly backfill data. You can do a full sync with your Meltano pipeline to get all of the data into the new format and then switch any downstream process to point at this new table. However, for many sources this may not be possible.
+
+Another option would be to stop your Airbyte pipeline and have the Meltano pipeline start at that point. For example, if the Airbyte pipeline stops on 2022-12-31 you could have the Meltano pipeline start at 2023-01-01 and write to a different table. You could then use a tool like dbt to transform both tables into a common format and then union them together into a single source.
+
+There are tradeoffs for both of these methods and you would have to determine where to best invest your time depending on your needs.
+
+If you have other ways to make this transition easier, we’d love to discuss them in Slack or Office Hours with you!
+
+### Do these connectors work with `meltano run` and `meltano elt`?
+
+Yes! Since the wrapper is based on the Meltano Singer SDK these connectors work just as they would a python-based Singer connector. You can [run](https://docs.meltano.com/reference/command-line-interface#run), [elt](https://docs.meltano.com/reference/command-line-interface#elt), [invoke](https://docs.meltano.com/reference/command-line-interface#invoke), [test](https://docs.meltano.com/reference/command-line-interface#test), and [configure](https://docs.meltano.com/reference/command-line-interface#config) them as you normally would.
+
+### Does this integration support stream maps for PII filtering/hashing?
+
+Yes! This is a unique feature of Meltano that is not found in as comprehensive manner in other modern EL tools. With stream maps users are able to hash, filter, duplicate, and alias any column or table during their extract. Because this Airbyte integration was built using the SDK, streams maps can be defined on the connector itself ([SDK docs](https://sdk.meltano.com/en/latest/stream_maps.html)), or they can be added separately as a mapper in between a tap and target executing using `meltano run`. See [our tutorial](https://docs.meltano.com/getting-started/part4#undefined) for an example.
```

Comment:
> found in as comprehensive manner in other

found in a comprehensive manner in other

Reply:
Fixed